### PR TITLE
benchmarks

### PR DESCRIPTION
**Pull request by Amuhar**

### CODE REVIEW

File: benchmarks/1-vu.script.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import http from "k6/http";
+import { check, sleep } from "k6";
+import { describe } from 'https://jslib.k6.io/expect/0.0.4/index.js';
```
Comment (infloop) [inline benchmarks/1-vu.script.js:3]: Can we avoid importing from remote https and import from local file?

File: README.md [hunk 2]
```diff
@@ -898,6 +898,11 @@ Check `sample.env` file.
 
 Pay attention that API by default running job for fetching and updating Validators. If you are not planning to use `validators` endpoints, you could disable this job by setting `VALIDATOR_REGISTRY_ENABLE=false`.
 
+## Benchmarks
+
+To see record output in file and send metrics to prometheus run command
+`K6_PROMETHEUS_RW_SERVER_URL=http://localhost:9090/api/v1/write k6 run --out json=benchmarks/output.json -o experimental-prometheus-rw --tag testid=<SET-HERE-A-UNIQUE-ID> benchmarks/200.script.js`
```
Comment (infloop) [inline README.md:904]: Do we need second example without prometheus rw here?

File: benchmarks/1-vu.script.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  // better to make it longer to cross with validators update
+  // we will need this benchmark to check how long 1 request is handled depends on kapi job execution
+  duration: "80s",
+  vus: 1
+};
+
+// Simulated user behavior
+export default function () {
+  // this endpoint process the
```
Comment (infloop) [inline benchmarks/1-vu.script.js:19]: Maybe there is a typo? Like `process the ...`

File: benchmarks/multiple-vu.script.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  duration: "60s",
+  vus: 10
+};
+
+// sometime get Unable to acquire a connection 
+// DriverException: Unable to acquire a connection 
+
+// Simulated user behavior
+export default function () {
+  // this endpoint process the
```
Comment (infloop) [inline benchmarks/multiple-vu.script.js:16]: The same as above

File: benchmarks/1-vu.script.js (hunk 1 above)
Comment (F4ever) [reply benchmarks/1-vu.script.js:3]: Wow, didn't know this could be done in ts

File: README.md (hunk 2 above)
Comment (Amuhar) [reply README.md:904]: added, and fixed description

File: benchmarks/1-vu.script.js (hunk 3 above)
Comment (Amuhar) [reply benchmarks/1-vu.script.js:19]: deleted

File: benchmarks/multiple-vu.script.js (hunk 4 above)
Comment (Amuhar) [reply benchmarks/multiple-vu.script.js:16]: deleted

**Review by infloop — APPROVED** [review]

**Pull request merged**